core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -2910,10 +2914,16 @@ private static long getMaxOfBlockAndFileSize(long blockSize, long fileSize) {
    * @return
    */
   public static String getBlockId(AbsoluteTableIdentifier identifier, String filePath,
-      String segmentId) {
+      String segmentId, boolean isUnmangedTable) {
     String blockId;
     String blockName = filePath.substring(filePath.lastIndexOf("/") + 1, filePath.length());
     String tablePath = identifier.getTablePath();
+
```

Comment:
this block id should be commonly constructed for unmanaged and managed case. As unmanged partition is also possible

Reply:
OK. Hanlded.